src/swarm/node/model/NeoNode.d

```diff
@@ -960,6 +961,48 @@ public class NodeBase ( ConnHandler : ISwarmConnectionHandler ) : INodeBase
         }
     }
 
+    /***************************************************************************
+
+        Unix domain socket connection handler, lists the credentials.
+
+        Params:
+            args          = command arguments
+            send_response = delegate to write to the client socket
+
+    ***************************************************************************/
+
+    private void handleListCredentials ( cstring args,
+        void delegate ( cstring response ) send_response )
+    in
+    {
+        assert(this.credentials_file);
```

Comment:
Shouldn't this be a `verify` in the method body?

Reply:
Right. I copy/pasted the existing handler which wasn't `verified` in v4.x.x branch.

Reply:
Ugh. We need to remember to test the merge commit - I need to import `swarm.util.Verify` here and I need to remember to remove it in v5.x.x, since we'll import both `ocean.core.Verify` and this one.

Reply:
Updated.

Reply:
Yeah, it's annoying :/